src/api/Flooq.Api/Services/IDataFlowService.cs

```diff
@@ -11,20 +11,60 @@ namespace Flooq.Api.Services;
 public interface IDataFlowService
 { 
   /// <summary>
-  /// Queries the all Data Flows from the Flooq database.
+  /// Queries the all DataFlows from the Flooq database.
   /// </summary>
-  /// <returns>all Data Flows</returns>
+  /// <returns>All DataFlows from the Flooq database.</returns>
   Task<ActionResult<IEnumerable<DataFlow>>> GetDataFlows();
 
-  Task<ActionResult<DataFlow>> GetDataFlow(Guid id);
+  /// <summary>
+  /// Finds a specific DataFlow in the Flooq database according to a unique identification.
+  /// </summary>
+  /// <returns>The identified DataFlow.</returns>
+  /// <param name="id">The value which is used for identifying the DataFlow.</param>
+  Task<ActionResult<DataFlow>> GetDataFlow(Guid? id);
   
+  /// <summary>
+  /// Sets a DataFlow entry according to the given EntityState.
+  /// For example if EntityState.Modified is used, the existing DataFlow entry will be overwritten by the new DataFlow.
+  /// </summary>
+  /// <param name="dataFlow">The new DataFlow.</param>
+  /// <param name="entityState">Defines the action of the new DataFlow entry.</param>
```

Comment:
Not quite clear what you mean here - is a new data flow created in any case? Otherwise the expression "The new DataFlow" would not be precise.